src/Profile/ProfileKeys.cpp

```diff
@@ -95,6 +95,14 @@ const char WatchedWaypointFile[] = "WatchedWPFile"; // pL
 const char LanguageFile[] = "LanguageFile"; // pL
 const char InputFile[] = "InputFile"; // pL
 const char PilotName[] = "PilotName";
+const char WeGlideEnabled[] = "WeGlideEnabled";
```

Comment:
These should be added in the commit which actually uses them. The commit splitted as it is serves no purpose. It's useless, it contains nothing.

Reply:
Merged with the next commit, where it is used